### ISSUE TITLE
Merge or squeeze logstash CI/CD pipelines

### ISSUE THREAD

**Issue by afeefghannam89** [NETWAYS/ansible-collection-elasticstack#172]
I think we can merge `logstash_defualt` and `logstash_pipeline` to one CI/CD pipeline. I think that the installation of logstash as standalone and testing his pipelines in one step will reduce the CI/CD running time and achieve two goals at once.

I am not sure if we can merge `logstash_run_logstash` with another CI/CD pipelines.

@widhalmt what do you think?

**Comment by widhalmt** [comment]
Good idea. I wanted to have both separated to test the role with as many defaults as possible. To show that the collection has a very low entry threshold. But since we have to set some variables nonetheless we can just merge one or both, too.

`logstash_run_logstash` was mostly introduced to check whether Logstash would start or not. So that we could have other pipelines without starting Logstash thus being faster. But I guess, that also lost its use now we change so much.

So yes, from my side, go ahead and merge them.
